DOCUMENTATION.md

```diff
@@ -10,83 +29,86 @@ If you want to read more into Flatseal's permissions, you can look at the [Permi
 
 If you want to read more into `flatpak override`, you can look at the [`flatpak-override` documentation](https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-override).
 
-
 ### Permissions
 
+This is a list of permissions that can be configured using Flatseal. The descriptions below assume that the options are enabled (toggled on) when it's a toggle.
+
+There is also a "`flatpak override` equivalent" row for those that want to get started on the CLI of Flatpak permissions.
+
 #### Share
 
 List of subsystems shared with the host system.
```

Comment:
All of these descriptions should be removed, to reduce the cognitive overload. The tables with descriptions and examples should be enough,

Reply:
I'm going to mark this as resolved. If you still don't like it, feel free to review that same section.